frontend/src/components/game/TimerPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Timer } from "./Timer";
+
+const TimerPage = () => {
+  const [start, setStart] = useState(false);
+  const [reset, setReset] = useState(false);
+  const [timerDisplay, setTimerDisplay] = useState("00:00");
```

Comment:
What do you think about making this into a number with centiseconds? And maybe we could extract the timeInCentiseconds into the TimerPage and format it outside of the timer as well? 

Just because right now if we want to use the final time for scoring, we'd have to parse the time from a string, which seems complicated

Reply:
Good idea i will do that